File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.kt
class Solution {
    fun solution(nums: IntArray): Int {
        var answer = 0
        var count = 0
        var sum = 0
        for(i in 0 until nums.size-2){
            for(j in i+1 until nums.size-1){
                for(k in j+1 until nums.size){
                    sum = nums[i]+nums[j]+nums[k]
                    for(a in 2..sum){
                        if(sum%a==0) {
                            count+=1
                        }
                    }
                    if(count==1) answer+=1
                    sum = 0
                    count = 0
                }
            }
        }
        return answer
    }
}

File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.py
def solution(nums):
    from itertools import combinations
    arr=list([sum(i) for i in combinations(nums,3)])
    answer = len(arr)
    print(arr)
    for i in arr:
        for j in range(2,int(i**0.5)+1):
            if i%j==0:
                answer-=1
                break
    return answer